src/main/java/com/money_account_service/utility/UserServiceClient.java

```diff
@@ -21,7 +21,7 @@ public class UserServiceClient {
 
     private final ObjectMapper objectMapper;
     @Value("${authorize.url}")
-    private String AUTHORIZE_URL;
+    private String AUTHORIZE_URL = "http://localhost:8081/authorize";
```

Comment:
Ideally, it should be injected to the class as parameter. Otherwise it shows troubles while testing.